src/components/Header.jsx

```diff
@@ -1,7 +1,54 @@
-import React from "react";
-
+import React, { useState } from "react";
+import { BiBell, BiSearch } from "react-icons/bi";
+import { MdClose, MdMenu } from "react-icons/md";
 const Header = () => {
-  return <div>Header</div>;
+  const [isNavOpen, setIsNavOpen] = useState(false);
+  return (
+    <header className="fixed z-50  left-0 right-0 top-0 text-dark justify-between bg-white py-4 px-6 shadow-md flex items-center">
+      <div className="left flex items-center gap-3">
+        {!isNavOpen ? (
+          <MdMenu
+            className="cursor-pointer rounded-full p-2 w-10 h-10 shadow-md  sm:hidden"
+            onClick={() => setIsNavOpen(true)}
+          />
+        ) : (
+          <MdClose
+            className="cursor-pointer rounded-full p-2 w-10 h-10 shadow-md sm:hidden"
+            onClick={() => setIsNavOpen(false)}
+          />
+        )}
+        <div className="site-title text-xl sm:inline-block hidden ">
+          <img src="" alt="" />
```

Comment:
Remove the empty image tag if you are not gonna use it. 
UI looks awesome. The rest of the code looks good to merge

Reply:
done